crates/storage/src/collections/bitstash/mod.rs

```diff
@@ -71,7 +71,8 @@ impl BitStash {
     /// in the `free` list.
     ///
     /// Returns the bit position of the first bit in the 256-bit chunk and not
-    /// the chunk position since that's what [`crate::collections::Bitvec::get_chunk`]
+    /// the chunk position since that's what
+    /// [`collections::Bitvec::get_chunk`][`crate::collections::Bitvec::get_chunk`]
```

Comment:
I think in this case it is okay to also leave away `collections::` prefix and start with the type name.